doc/nrf/releases_and_maturity/releases/release-notes-3.2.0.rst

```diff
@@ -0,0 +1,1250 @@
+    See the `migration guide <Migration guide for nRF Connect SDK v3.2.0_>`_ for more information.
+  * By moving code from :file:`samples/matter/common/src/bridge` to :file:`applications/matter_bridge/src/core` and :file:`applications/matter_bridge/src/ble` directories.
+  * The Identify cluster implementation in the application to use the code-driven approach instead of the zap-driven approach.
+  * The default number of Bluetooth Low Energy connections that can be selected using the Kconfig configuration from ``10`` to ``8`` for the Matter bridge over Thread configuration.
+
+nRF5340 Audio
+-------------
+
+* Added:
+
+  * A way to store servers in RAM on the unicast client (gateway) side.
+    The storage does a compare on server address to match multiple servers in a unicast group.
+    This means that if another device appears with the same address, it is treated as the same server.
+  * Experimental support for stereo in :ref:`unicast server application<nrf53_audio_unicast_server_app_configuration_stereo>`.
+  * The :ref:`Audio application API documentation <audio_api>` page.
+  * The :ref:`nrf53_audio_app_config_audio_app_options` page.
+  * The API documentation in the header files listed on the :ref:`audio_api` page.
+  * Ability to connect by address as a unicast client.
+
+* Updated:
+
+  * The unicast client (gateway) application has been rewritten to support N channels.
+  * The unicast client (gateway) application now checks if a server has a resolvable address.
+    If this has not been resolved, the discovery process starts in the identity resolved callback.
+  * The power measurements to be disabled by default in the default debug versions of the applications.
+    To enable power measurements, see :ref:`nrf53_audio_app_configuration_power_measurements`.
+  * The audio application targeting the :zephyr:board:`nrf5340dk` to use pins **P1.5** to **P1.9** for the I2S interface instead of **P0.13** to **P0.17**.
+    This change avoids conflicts with the onboard peripherals on the nRF5340 DK.
+  * The documentation pages with information about the :ref:`SD card playback module <nrf53_audio_app_overview_architecture_sd_card_playback>` and :ref:`how to enable it <nrf53_audio_app_configuration_sd_card_playback>`.
+  * The API documentation in the header files listed on the :ref:`audio_api` page.
+
+* Removed the LC3 QDID from the :ref:`nrf53_audio_feature_support` page.
+  The QDID is now listed in the `nRF5340 Bluetooth DNs and QDIDs Compatibility Matrix`_.
+
+nRF Desktop
+-----------
+
+* Added:
+
+  * Support for the ``nrf54h20dk/nrf54h20/cpuapp`` board target configurations, aligned with the IronSide SE architecture.
+    These configurations use the MCUboot bootloader in direct-XIP mode and a merged image slot that combines both the application and radio core images.
+    They provide the same feature set as the SUIT-based configuration introduced in |NCS| v3.0.0, except for the change in the DFU solution.
```

Comment:
The SUIT-based configurations weren't introduced in |NCS| v3.0 - they were simply released as part of the NCS 3.0. The SUIT configurations were available in the nRF Desktop since v2.7.0. I was just referring to the last release in which they worked correctly

Here is my original suggestion for this sentence:

_(...), they provide the same feature set as the SUIT-based configuration released in |NCS| v3.0.0._